tests/eval_tests/utils.py

```diff
@@ -101,13 +143,12 @@ def save_plot(fig, filename, dpi=300, bbox_inches='tight'):
     if not filename.lower().endswith(('.jpg', '.jpeg')):
         filename += '.jpg'
     filename = filename.replace(" ", "_")
+    filename = filename.replace("/", "_")
```

Comment:
Will the file name contain `/`? it should just be what ever the `title` param in `add_plot()` is right?

Reply:
I don't think that you want the full path in the file name. Is the a reason to do it that way?

Reply:
@Ibrahim-Haroon Why does the file name include the filepath? Can this be removed?